src/screens/About.js

```diff
@@ -28,6 +29,16 @@ export class About extends React.Component {
     },
   }));
 
+  touchCount = 0;
+
+  onPressLogo = () => {
+    // Method created to test error handling
+    this.touchCount += 1
+    if (this.touchCount === 10) {
+      throw new Error('Hathor test error.')
```

Comment:
It might open some "Dev tools" where we can include the "Copy data" you were talking about. Even though, I'm not sure we should have "copy data" in the first place.

Reply:
Yeah, I agree with you. This copy data may not be good, I think we should go without it, at least for now.